js/index.js

```diff
@@ -0,0 +1,89 @@
+$(document).ready(function() {
+
+	var secretNumber;
+	var userGuess;
+	var numberOfGuessesRemaining = 5;
+
+	secretNumber = getRandomIntInclusive();
+
+// app develops secret number in var secretNumber
+// whole number
+// between 1 and 100
+	function getRandomIntInclusive(min, max) {
+  		return Math.floor(Math.random() * (100 - 1 + 1)) + 1;
+	}
+
+// pull userGuess
+	$("#guessButton").click(compare);
+
+// compare userGuess with secretNumber
+	function compare(event) {
+		event.preventDefault();
+		userGuess = parseInt($("#guess").val());
+		console.log(secretNumber);
+		console.log(userGuess);
+
+// reduce numberOfGuessesRemaining
+		numberOfGuessesRemaining = numberOfGuessesRemaining - 1;
+		$("#numberOfGuessesRemaining").text(numberOfGuessesRemaining);
+
+		if (isNaN(userGuess) || userGuess > 100 || userGuess < 1) {
+      		// alert("Please enter an amount between 1 and 100!");
+      		$(".message").text("Please enter an amount between 1 and 100!");
+    	}
+// if the same, alert/type "Success!" in notification
+		else if (userGuess === secretNumber) {
+			// alert("Success!");
+			$(".message").text("Success!");
+			$("body").attr("class", "success");
+		}
+// depending on secretNumber, if userGuess is higher, notify "Guess lower number!"
+		else if (userGuess > secretNumber) {
+			// alert("Guess lower number!");
+			$(".message").text("Guess a lower number!");
+			$("body").attr("class", "error");
+		}
+// depending on secretNumber, if userGuess is lower, notify "Guess higher number!"
+		else if (userGuess < secretNumber) {
+			// alert("Guess higher number!");
+			$(".message").text("Guess a higher number!");
+			$("body").attr("class", "error");
+		}
+		else {
```

Comment:
:hammer: @daniwhkim  Looks like your logic doesn't require an else statement. There's nothing wrong with leaving it off, so I would remove this.